cbmackintosh/hooloo
# Feature/site structure

**Clearly and concisely describe the feature.**
- Rudimentary login system utilizes localStorage to keep user logged in
- Establishes basic site structure and navigation
- Some initial styling and network requests to provide relevant content

**Problems Encountered & Resulting Solutions**
- Nav bar exhibits some strange behavior - vertical offset, covers page content. Need to normalize CSS?

**How should this be tested?**
- Clone the repository
- `npm install`
- `npm start`
- Login to the application with email: alan@turing.io, password: promisedotyall
- Refresh to ensure page persistence
- Navigate to account page to verify correct user info is displayed
- Check dev tools to ensure all relevant data is returned from the network request
